test/src/main/configurations/HTTP/ConfigurationWebServiceListenerSender.xml

```diff
@@ -2,7 +2,7 @@
 	<adapter name="WebServiceListener" description="Test the functioning of the WebServiceListener">
 		<receiver>
 			<listener className="nl.nn.adapterframework.http.WebServiceListener"
-				name="urn:ws" />
+				name="urn:ws" serviceNamespaceURI="urn:ws" />
```

Comment:
Could you revert the changes made in this file, as we explicitly test the endpoints here. The other 2 files are ok!